src/components/BooksForm.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { addBook } from '../actions';
+
+function BooksForm({ dispatch }) {
```

Comment:
Let place container and presentational components. Also, let's make use of es6 syntax.

Reply:
Hi Team
I think we still need to address this. `BooksForm` should also be in the container folder.
![image](https://user-images.githubusercontent.com/31897434/113963195-15ae0180-9821-11eb-81f7-7fc5a2bef189.png)